src/features/submitCode/gitPush/ui/GitPushDialog.tsx

```
@@ -14,13 +14,14 @@ import { Tooltip, TooltipContent, TooltipTrigger } from '@/components/ui/tooltip
 import { useEffect, useState } from 'react';
 import { Select } from '@/shared/ui/select/Select';
 
-interface GitPushDialogProps {
-  githubUrl: string | null;
-}
-export default function GitPushDialog({ githubUrl }: GitPushDialogProps) {
+interface GitPushDialogProps {}
+
+export default function GitPushDialog({}: GitPushDialogProps) {
```

Biome (2.1.2)

[error] Unexpected empty object pattern.



(lint/correctness/noEmptyPattern)